Mathlib/Topology/Category/TopCat/Limits/Konig.lean

```diff
@@ -1,17 +1,17 @@
 /-
-Copyright (c) 2021 Kyle Miller. All rights reserved.
+Copyright (c) 2017 Scott Morrison. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Kyle Miller
+Authors: Patrick Massot, Scott Morrison, Mario Carneiro, Andrew Yang
```

Comment:
@kmill, do you know which copyright is correct? The new version is what we have in mathlib3.

Reply:
I fixed the copyright notice in mathlib4 (check the blame) but I didn't backport it to mathlib3.